pkgs/development/compilers/gambit/build.nix

```diff
@@ -57,7 +56,7 @@ gccStdenv.mkDerivation rec {
     # "--enable-coverage"
     # "--enable-inline-jumps"
     # "--enable-char-size=1" # default is 4
-  ];
+  ] ++ lib.optional (!gccStdenv.isDarwin) "--enable-poll";
```

Comment:
Maybe also add the reason for disabling this.

Reply:
done, thanks